app/models/study_processor.rb

```diff
@@ -0,0 +1,16 @@
+class StudyProcessor
+    def self.process
+        saved_nil = []
+        StudyJsonRecord.all.each do |study|
```

Comment:
This will load all the 500k studies, instead of doing this, we want to use activerecord like `where('saved_study_at NULL OR updated_at > saved_atudy_at)`

Reply:
> This will load all the 500k studies, instead of doing this, we want to use activerecord like `where('saved_study_at NULL OR updated_at > saved_atudy_at)`

Thank you for reviewing, method is fixed.